AltaPayApi/AltaPayApi/Service/Requests/ReserveRequest.cs

```diff
@@ -10,6 +10,13 @@ public class ReserveRequest
 		public string ShopOrderId {get; set;}
 		public Amount Amount { get; set; }
 		public AuthType PaymentType { get; set; }
+		public AgreementType? AgreementType { get; set; }
+		public string AgreementId { get; set; }
+		public AgreementUnscheduledType? AgreementUnscheduledType { get; set; }
+		public string AgreementExpiry { get; set; }
+		public string AgreementFrequency { get; set; }
+		public string AgreementNextChargeDate { get; set; }
+		public string AgreementAdminUrl { get; set; }

```

Comment:
Why not using a complex object, like the PaymentRequestConfig

Reply:
Great idea, done, thank you.